agent/bench-scripts/pbench-uperf

```diff
@@ -369,12 +381,29 @@ while true; do
 		;;
 	esac
 done
+
+if [[ ${do_pre_check_only} -eq 1 ]]; then
+	local_pre_check ${@}
+	exit ${?}
+fi
+
 verify_common_bench_script_options $tool_group $sysinfo
 
 clients=${clients:-${clients_def}}
 servers=${servers:-${servers_def}}
 
 if [[ "${postprocess_only}" != "y" ]]; then
+	ver="$(pbench-config version ${benchmark})"
+	if [[ -z "${ver}" ]]; then
+		error_log "${script_name}: ${benchmark} 'version' is missing in config file"
+		exit 1
+	fi
+	match="$(pbench-config match ${benchmark})"
+	if [[ -z "${match}" ]]; then
+		error_log "${script_name}: ${benchmark} 'match' is missing in config file"
+		exit 1
+	fi
```

Comment:
I prefer the friendlier messages which you used in `agent/bench-scripts/pbench-fio`:
```
error_log "${script_name}: 'version' is missing in the [${benchmark}] section of the config file"
```

Reply:
(f)io has (f)riendly messages, and (u)perf has (u)nfriendly messages.  Symmetric, right?